mantis-control-plane/mantis-control-plane-server/src/test/java/io/mantisrx/master/api/akka/payloads/ResourceClustersPayloads.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.mantisrx.master.api.akka.payloads;
+
+public class ResourceClustersPayloads {
+    public static final String CLUSTER_ID = "mantisResourceClusterUT1";
+
+    public static final String RESOURCE_CLUSTER_CREATE = "{\"clusterId\":\"mantisResourceClusterUT1\",\"clusterSpec\":"
```

Comment:
More of a personal preference than anything, but these may be painful to modify for testing if the data structure changes in any sort of meaningful way. Some kind of structure that gets serialized, or even a well formatted ".json" file that is read in would be easier to edit.

Reply:
+1

Reply:
+1. Moved to a resource JSON file.